applications/MeshingApplication/custom_processes/metrics_error_process.cpp

```diff
@@ -138,22 +141,24 @@ void MetricErrorProcess<TDim>::CalculateMetric()
     }
 
     // Iteration over all nodes
-    const int num_nodes = static_cast<int>(r_nodes_array.size());
-    KRATOS_DEBUG_ERROR_IF(num_nodes == 0) <<  "ERROR:: Empty list of nodes" << std::endl;
+    KRATOS_DEBUG_ERROR_IF(r_nodes_array.size() == 0) <<  "ERROR:: Empty list of nodes" << std::endl;
+
+    // Auxiliar variables
+    const auto average_nodal_h = mAverageNodalH;
+    const auto echo_level = mEchoLevel;
```

Comment:
why is this necessary?
Cannot you capture those too?

Reply:
or use `this`?

Reply:
...that's genious...